docs/product/security/ip-ranges.mdx

```diff
@@ -13,10 +13,13 @@ apply to self-hosted or single tenant.
 
 ## Dashboard and API
 
-Sentry's dashboard and API are both served from a single IP address for all web traffic. The IP address is:
+Sentry's dashboard and API are both served from different of domains depending
+on your organization's data storage location. The IP addresses are:
 
 ```
-35.186.247.156/32
+sentry.io 35.186.247.156/32
+us.sentry.io 35.186.247.156/32
+de.sentry.io 34.36.122.224/32
```

Comment:
We have two addresses reserved for `de.sentry.io`, to facilitate maintenance/upgrades in the future:
```suggestion
de.sentry.io 34.36.122.224/32, 34.36.87.148/32
```